stdpopsim/catalog/ApiMel/demographic_models.py

```diff
@@ -0,0 +1,26 @@
+# import msprime
```

Comment:
We should probably leave this file out of the initial species addition I think - we can follow up with other PRs for demographic models.

Reply:
Yes, agreed, we are still working on the models anyway

Reply:
@jeromekelleher , it took me some time to come back to this but I have corrected some mistakes in the species.py file, added some references and also removed the demographic model from the git. Could you please revise this version? Thanks!